core/benches/arrays.rs

```diff
@@ -32,6 +35,7 @@ fn ncl_random_array(len: usize) -> String {
     String::from_utf8(out).unwrap()
 }
 
+#[cfg(not(feature = "benchmark-ci"))]
```

Comment:
If the whole `ncl_bench_group!` is gated behind the feature, is there a reason why we need to disable individual benchmark with a `#[cfg(not(...))]` as well ?

Reply:
Without configuring out those things, CI failed with unused code warnings.